frontend/src/pages/ActivityReport/Pages/components/Objective.js

```diff
@@ -127,6 +127,33 @@ export default function Objective({
     defaultValue: objective.files || [],
   });
 
+  const {
+    field: {
+      onChange: onChangeUseIpdCourses,
+      onBlur: onBlurUseIpdCourses,
+      value: objectiveUseIpdCourses,
+      name: objectiveUseIpdCoursesInputName,
+    },
+  } = useController({
+    name: `${fieldArrayName}[${index}].useIpdCourses`,
+    defaultValue: !!(objective.courses && objective.courses.length) || false,
+  });
+
+  const {
+    field: {
+      onChange: onChangeIpdCourses,
+      onBlur: onBlurIpdCourses,
+      value: objectiveIpdCourses,
+      name: objectiveIpdCoursesInputName,
+    },
+  } = useController({
+    name: `${fieldArrayName}[${index}].courses`,
+    defaultValue: objective.courses || [],
+    rules: {
+      required: objectiveUseIpdCourses === true,
```

Comment:
Not sure if we want validation that checks length like we fixed on my pr.

Reply:
ok